src/Command/CoasterCloseCommand.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Command;
+
+use App\Entity\Coaster;
+use App\Entity\Status;
+use App\Repository\CoasterRepository;
+use App\Repository\StatusRepository;
+use DateTime;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Notifier\ChatterInterface;
+use Symfony\Component\Notifier\Message\ChatMessage;
+
+class CoasterCloseCommand extends Command
+{
+    protected static $defaultName = 'coaster:close';
+
+    public function __construct(
+        private readonly CoasterRepository      $coasterRepository,
```

Comment:
please apply php-cs-fixer rules (config file at root of the project)

Reply:
do you need help?

Reply:
I tried to launch the `.php-cs-fixer.dist.php` script (from terminal and Docker container), with `php .php-cs-fixer.dist.php fix src`, but the class PhpCsFixer\Config is not found.

Reply:
I will merge this for now, and explain you how to set up php-cs-fixer later on